packages/ckeditor5-dev-tests/lib/tasks/runmanualtests.js

```diff
@@ -42,9 +42,10 @@ const transformFileOptionToTestGlob = require( '../utils/transformfileoptiontote
 module.exports = function runManualTests( options ) {
 	const log = logger();
 	const buildDir = path.join( process.cwd(), 'build', '.manual-tests' );
-	const files = ( options.files && options.files.length ) ? options.files : [ '*' ];
-	const sourceFiles = files
-		.flatMap( file => transformFileOptionToTestGlob( file, true ) )
+	const isFilesFlagProvided = ( options.files && options.files.length );
+	const files = isFilesFlagProvided ? options.files : [ '*', 'ckeditor5' ];
+	const dedupedFileTestGlobs = [ ...new Set( files.flatMap( file => transformFileOptionToTestGlob( file, true ) ) ) ];
```

Comment:
Good catch with that 👍🏻 Previously, calling `yarn manual -f table,utils,table` would produce duplicated entries. Now it will be fixed.

Maybe it would be good to add a small test for that?

Reply:
Added a test for this case.